Fix tests\P0067R5_charconv\test.cpp for C++20 addition to `std::chrono`

Potential good first issue.

C++ 20 adds `std::chrono::last` which is a global variable. The tests treat the warning C4459 hiding the declaration of a global as an error.

Currently `tests\P0067R5_charconv\test.cpp` has `using namespace std::chrono;`

Which means the following trigger this warning.

https://github.com/microsoft/STL/blob/31419650d472932dd24b9c5602707b0454f61fb2/tests/std/tests/P0067R5_charconv/test.cpp#L147

https://github.com/microsoft/STL/blob/31419650d472932dd24b9c5602707b0454f61fb2/tests/std/tests/P0067R5_charconv/test.cpp#L633

https://github.com/microsoft/STL/blob/31419650d472932dd24b9c5602707b0454f61fb2/tests/std/tests/P0067R5_charconv/test.cpp#L680

I suppose the simplest solution is to just rename them.

--- a/tests/std/tests/P0067R5_charconv/test.cpp
+++ b/tests/std/tests/P0067R5_charconv/test.cpp
@@ -1070,7 +1070,8 @@ int main(int argc, char** argv) {
     all_floating_tests(mt64);
 
     const auto finish  = chrono::steady_clock::now();
-    const long long ms = chrono::duration_cast<milliseconds>(finish - start).count();
+    const long long ms = chrono::duration_cast<chrono::milliseconds>(finish - start).count();
+
 
     puts("PASS");
     printf("Randomized test cases: %u\n", PrefixesToTest * Fractions);

--- a/tests/std/tests/P0067R5_charconv/test.cpp
+++ b/tests/std/tests/P0067R5_charconv/test.cpp
@@ -1072,7 +1072,6 @@ int main(int argc, char** argv) {
     const auto finish  = chrono::steady_clock::now();
     const long long ms = chrono::duration_cast<chrono::milliseconds>(finish - start).count();
 
-
     puts("PASS");
     printf("Randomized test cases: %u\n", PrefixesToTest * Fractions);
     printf("Total time: %lld ms\n", ms);

--- a/tests/std/tests/P0067R5_charconv/test.cpp
+++ b/tests/std/tests/P0067R5_charconv/test.cpp
@@ -46,7 +46,6 @@
 #include <vector>
 
 using namespace std;
-using namespace std::chrono;
 
 void initialize_randomness(mt19937_64& mt64, const int argc, char** const argv) {
     constexpr size_t n = mt19937_64::state_size;
@@ -1060,7 +1059,7 @@ void all_floating_tests(mt19937_64& mt64) {
 }
 
 int main(int argc, char** argv) {
-    const auto start = steady_clock::now();
+    const auto start = chrono::steady_clock::now();
 
     mt19937_64 mt64;
 
@@ -1070,8 +1069,8 @@ int main(int argc, char** argv) {
 
     all_floating_tests(mt64);
 
-    const auto finish  = steady_clock::now();
-    const long long ms = duration_cast<milliseconds>(finish - start).count();
+    const auto finish  = chrono::steady_clock::now();
+    const long long ms = chrono::duration_cast<milliseconds>(finish - start).count();
 
     puts("PASS");
     printf("Randomized test cases: %u\n", PrefixesToTest * Fractions);

